WebContent/js/components/JobInstance.js

```diff
@@ -95,18 +95,27 @@ class JobInstance extends React.Component {
         const statusStyleActive = { display: 'inline' };
         const statusStyleAbend = { color: 'red', display: 'inline' };
         const statusStyleComplete = { color: 'grey', display: 'inline' };
-        if (job.get('returnCode')) {
-            if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')) {
-                return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+        const errorReturnCode = ['abend', 'jcl error', 'sys fail', 'conv error', 'sec error'];
```

Comment:
These variable names should be in plural form as it's an array of multiple, so errorReturnCode**s** and completeReturnCode**s**